### PR TITLE
Github package registry & gradle

### PR DESCRIPTION
**Pull request by pastjean**
- Move to gpr
- Move to gradle
- Update feign

See: https://github.com/pleo-io/feign-jaxrs2/packages

https://app.clubhouse.io/pleo/story/17916/jvm-from-artifactory-to-github-package-repository

### CODE REVIEW

**Comment by pastjean** [comment]
@GuiSim do you remember why we created that dependency. It seems that feign now supports jaxrs2 natively 👍 https://github.com/OpenFeign/feign/tree/master/jaxrs2

**Comment by GuiSim** [comment]
> @GuiSim do you remember why we created that dependency. It seems that feign now supports jaxrs2 natively 👍 https://github.com/OpenFeign/feign/tree/master/jaxrs2

@pastjean 

Yeah it was Emile. I think either JAXRS2 was not supported at all or only partially supported.
The annotations in clients didn't behave exactly the way we wanted? Pretty sure if we diff our JAXRS Contract VS the OpenFeign one there will be differences. Ours is based heavily on the JAXRS1 Contract that was available in OpenFeign.

TL;DR: We could test this, if the OpenFeign JAXRS2 Contract works, we don't need our custom class.

**Comment by pastjean** [comment]
Ill look into testing this , im interested thx for the deep knowledge @GuiSim

**Comment by pastjean** [comment]
I'm merging it so my work is not lost, but ... it's gonna be. since this repo alltogether is not needed!!!
